runelite-api/src/main/java/net/runelite/api/SpritePixels.java

```diff
@@ -51,4 +51,10 @@
 	 * Width and Height must match
  	 */
 	void toBufferedImage(BufferedImage img);
+
+	void setOutline(int outlineColor);
```

Comment:
Unused. I'm not too sure but I think @Adam- prefers to not import anything we won't be using so you could drop the whole renaming of the method in `runescape-client` and it's imports in the API.